### DISCUSSION TITLE
Configurable version and api path

### DISCUSSION THREAD

**Discussion by uyab** [dedoc/scramble#35] [Ideas]
Version and api path are currently hardcoded:
```php
   private function makeOpenApi()
    {
        $openApi = OpenApi::make('3.1.0')
            ->setComponents($this->transformer->getComponents())
            ->setInfo(InfoObject::make(config('app.name'))->setVersion('0.0.1'));

        $openApi->addServer(Server::make(url('/api')));

        return $openApi;
    }
```

Will you accept a PR that move that value into config file?

**Comment by romalytvynenko** [comment]
Hey @uyab 

Thanks for asking. 

I'll be glad to accept a PR for this. Thank you. 

Please also check out the previous related issue with suggested config structure. This is what I was going to go with: https://github.com/dedoc/scramble/issues/23

Thanks.

**Comment by romalytvynenko** [comment]
Hey @uyab 

Staring from `0.4.0` you can do that.

You can customize API base URL and info. Here is a docs: https://scramble.dedoc.co/usage/info

🙌
